src/subDivBoxTree.py

```diff
@@ -0,0 +1,148 @@
+import time
+
+import numpy as np
+
+from bounds import BBox
+from rayTracing import dmnsn_aabb
+
+
+class SubDivBoxTree(dmnsn_aabb):
```

Comment:
would you please add some information about this class, like what type of `SubDiv` tree it is? how to add new nodes/leafs (that might be outside of original bounds), scenarios where it might be used, and how, ..... is it possible to make it hold different type of data?

...questions of that kind

Reply:
Done by adding some comments with information of SubDixBoxTree class.